tests/FSharp.Compiler.ComponentTests/FSharpChecker/CommonWorkflows.fs

```diff
@@ -89,3 +89,13 @@ let ``Changes in a referenced project`` () =
         saveFile "Library"
         checkFile "Last" expectSignatureChanged
     }
+
+[<Fact>]
+let ``Language service works if the same file is listed twice`` () = 
+    let file = sourceFile "First" []
+    let project =  SyntheticProject.Create(file)
+    project.Workflow {       
+        checkFile "First" expectOk
+        addFileAbove "First" file
+        checkFile "First" (expectSingleWarningAndNoErrors "Please verify that it is included only once in the project file.")
```

Comment:
Should we check the exact message to ensure the numbers are correct?

Reply:
I see the numbers as a cosmetic detail which is only a small guide, not and essential part of this feature.